src/woo_search/search_index/documents/publications.py

```diff
@@ -39,3 +45,23 @@ class Document(ES_Document):
 
     class Index:
         name = "document"
+
+
+class Publication(ES_Document):
+    uuid: M[str] = mapped_field(Text(required=True))
+    publisher: M[PublisherType] = mapped_field(Object(Publisher, required=True))
+    informatie_categorieen: M[List[InformatieCategorieType]] = mapped_field(
```

Comment:
Ik zie geen informatie_categorieen op `class Document(ES_Document)`. Hoe gaat dat werken als je uiteindelijk filtert op informatiecategorie? Ik vermoed dat je dan ook de documenten wil zien die bij een publicatie horen die die informatiecategorie heeft. Wat mij betreft slaan we alle velden van de publicatie ook plat op de documenten.